sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java

```diff
@@ -434,6 +434,27 @@ protected double makePositive(double dValue, String message) {
         }
     }
 
+    protected boolean isNotNegative(Double dValue) {
```

Comment:
Again the return type should be void since this will never return false.  The method name should be "validateNegativeValue" or something along those lines

Reply:
also please add comments to all methods